src/libs/ajax/workspaces/providers/ExportWorkflowToWorkspaceProvider.ts

```diff
@@ -4,7 +4,7 @@ import { MethodConfiguration, MethodRepoMethod } from 'src/libs/ajax/workspaces/
 import { WorkspaceInfo } from 'src/workspaces/utils';
 
 export interface ExportWorkflowToWorkspaceProvider {
-  export: (destWorkspace: WorkspaceInfo, destWorkflowName: string, options?: AbortOption) => Promise<void>;
+  export: (destWorkspace: WorkspaceInfo, destWorkflowName: string, options?: AbortOption) => Promise<any>;
```

Comment:
You could make this `Promise<Response>` for consistency since I believe that the Ajax response is being returned.

Reply:
The `export` function of the provider created by the other factory method in this file returns `Promise<any>`, and the value that promise resolves to does not appear to be a `Response`.